src/live.asp.net/TagHelpers/BodyTagHelper.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.AspNet.Razor.Runtime.TagHelpers;
+
+namespace live.asp.net.TagHelpers
+{
+    public class BodyTagHelper : TagHelper
+    {
+        public override void Process(TagHelperContext context, TagHelperOutput output)
+        {
+            base.Process(context, output);
+
+            if (!output.Content.GetContent().Contains("$(\"p[data-hidden-value]\"))")
+                && output.Content.GetContent().Contains("click-to-show"))
+            {
+                const string script = @"<script>
```

Comment:
Please don't use `const` for things like this, it forces it to get compiled in to the assembly so we tend to not use it for anything that could change in the future. I'd prefer not to include JS inline in the C# at all but rather put it in a separate JS file that is compiled in as an embedded resource then read out into a static member just once (see https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs#L334)